### ISSUE TITLE
Add global dataset

### ISSUE THREAD

**Issue by embreinhardt** [nytimes/covid-19-data#67]
Also making public the global dataset of case and death timeseries for each country would be hugely valuable.

https://www.nytimes.com/interactive/2020/world/coronavirus-maps.html

Thanks!

**Comment by aanand08** [comment]
@embreinhardt , a good source of global data is https://coronavirus.jhu.edu/map.html and the github link for data is at the bottom of the page. JHU database comes with its challenges of data structure changing over time and potential inaccuracies in cumulative case counts.

**Comment by TGasinski** [comment]
Just wanted to agree with @embreinhardt that a similar dataset for other countries would be incredibly helpful!

**Comment by j-p-c** [comment]
https://opendata.ecdc.europa.eu/covid19/casedistribution/csv/ is useful as a world dataset

**Comment by cipriancraciun** [comment]
@aanand08 the JHU dataset has quite a lot of data inconsistencies (let alone formatting issues);  there are countless open issues, which are left unattended to.

For example, I've reported that on a given day all the values for France were wrongly put as French Polynesia, and although the issue isn't solved yet, they've closed the issue.
* https://github.com/CSSEGISandData/COVID-19/issues/2094#issuecomment-611744632

----

@j-p-c the ECDC dataset seems to have less data issues, but on the other side their reporting is less transparent.  They do point out their method:
* https://www.ecdc.europa.eu/en/covid-19/data-collection

However, if there is an issue with the numbers where do we report the inconsistency?

That is why I think a journalistic approach would be better, or at least would provide an alternative view.

----

BTW, I've started a few weeks ago a project to homogenize (and augment) the JHU, NY Times and ECDC datasets, and its available at: https://github.com/cipriancraciun/covid19-datasets

One could easily write an SQL query (or Excel lookup) to check the differences between all the datasets.

From what I've seen the differences are quite large.
